### DISCUSSION TITLE
org-modern-mode fontifies heading number from org-num-mode when heading has TODO tags

### DISCUSSION THREAD

**Discussion by drghirlanda** [minad/org-modern#190] [Ideas]
It looks like org-modern-mode fontifies the numbers inserted by org-num-mode in headings with a TODO tag. For example:

    * TODO Heading

Renders like this with both org-num-mode and org-modern-mode on:

![org-modern-issue](https://github.com/minad/org-modern/assets/5239356/e4f35ec0-1aa2-4e70-855e-803e41e04a69)

You can see that the 18 is smaller font and the color and background are changed. This happens regardless of which mode I activate first.

**Comment by minad** [comment]
Thanks for the report. This means that org-num-mode is incompatible with org-modern-mode. Tbh I wasn't even aware of org-num-mode. Patches improving the situation are welcome.

**Reply by drghirlanda** [reply]
Thanks, I'll try to look into that. I'm guessing org-num-mode uses an overly that we can look at to determine where the tag actually starts. I'm also guessing that org-modern-mode just goes to the beginning of the line past the stars, and this is too much if org-num-mode is active.
